src/EnergyPlus/ElectricPowerServiceManager.cc

```diff
@@ -2359,7 +2359,7 @@ DCtoACInverter::DCtoACInverter(EnergyPlusData &state, std::string const &objectN
     : aCPowerOut_(0.0), aCEnergyOut_(0.0), efficiency_(0.0), dCPowerIn_(0.0), dCEnergyIn_(0.0), conversionLossPower_(0.0), conversionLossEnergy_(0.0),
```

Comment:
I thought we were moving away from these types of constructors. Is that migration over?